mock_test.go

```diff
@@ -470,3 +469,73 @@ func TestConsumerBackoffDisconnect(t *testing.T) {
 		}
 	}
 }
+
+func TestConsumerPause(t *testing.T) {
+	msgIDGood := MessageID{'1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 'a', 's', 'd', 'f', 'g', 'h'}
```

Comment:
not an expected message id, as it has characters that are not hex ... not sure if it matters at all

very minor idea, alternate way to write this:

```go
	var msgId MessageID
	copy(msgId[:], []byte("0123456789abcdef"))
```

Reply:
copied from previous tests and yes, it doesn't matter